skll/utilities/generate_predictions.py

```diff
@@ -169,8 +200,34 @@ def main(argv=None):
                                                     label_col=args.label_col,
                                                     id_col=args.id_col)
             feature_set = reader.read()
-            for pred in predictor.predict(feature_set):
-                print(pred)
+            preds = predictor.predict(feature_set)
+            header = predictor.output_file_header
+
+            if args.output_file is not None:
+                with open(args.output_file, "a") as fout:
```

Comment:
I am not a fan of C-style file handle names:
```suggestion
                with open(args.output_file, "a") as outputfh:
```